CONTRIBUTING.md

```diff
@@ -199,7 +199,7 @@ Rust code follows the [Rust style guide](https://github.com/rust-lang-nursery/fm
 After installation, you can run it with
 ```$sh
 $ cd exonum-java-binding/core/rust
-$ cargo fmt --all -- --check
+$ cargo +stable fmt --all -- --check
```

Comment:
Is it needed here? (I used to use the default one)

Reply:
We actually use stable one on Travis. The output of formatter is pretty stable now, it looks it doesn't change much between versions, but I decided to clarify to avoid "Travis is red while I checked locally"